src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,153 @@
 package core.basesyntax;
 
+//import java.util.HashMap;
+import java.util.Objects;
+import java.util.Random;
+
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public double EXPANSION_COEFFICIENT = 2;
```

Comment:
Constants should have static final modifiers. So please add these modifiers here.
Ref: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names